blueprints/cloud-operations/network-dashboard/src/README.md

```diff
@@ -2,9 +2,10 @@
 
 This tool constitutes the discovery and data gathering side of the Network Dashboard, and can be used in combination with the related [Terraform deployment examples](../), or packaged in different ways including standalone manual use.
 
-- [Quick Usage Example](#quick-usage-example)
-- [High Level Architecture and Plugin Design](#high-level-architecture-and-plugin-design)
-- [Debugging and Troubleshooting](#debugging-and-troubleshooting)
+- [Network Dashboard Discovery Tool](#network-dashboard-discovery-tool)
```

Comment:
This heading in the ToC and the ToC indents do not match the hierarchy in the doc

Reply:
fixed indentation!